x-pack/plugins/security_solution/public/management/pages/policy/view/protection_updates/protection_updates_layout.tsx

```diff
@@ -247,10 +249,25 @@ export const ProtectionUpdatesLayout = React.memo<ProtectionUpdatesLayoutProps>(
               }
             )}
           >
-            {i18n.translate('xpack.securitySolution.endpoint.protectionUpdates.manifestOutdated', {
-              defaultMessage:
-                'Manifest is older than 30 days. Recommended to update the manifest or enable "Update manifest automatically".',
-            })}
+            <FormattedMessage
+              id="xpack.securitySolution.endpoint.protectionUpdates.manifestOutdated"
+              defaultMessage="Your protections artifacts have not been updated in over 30 days. We strongly recommend keeping these up to date ensure the highest level of security for your environment.{break}Note: After 18 months, protections artifacts will expire and cannot be rolled back. {learnMore}"
```

Comment:
```suggestion
              defaultMessage="Your protection artifacts have not been updated in over 30 days. We strongly recommend keeping these up to date to ensure the highest level of security for your environment.{break}Note: After 18 months, protections artifacts will expire and cannot be rolled back. {learnMore}"
```

Reply:
@caitlinbetz ^^^

Reply:
I was going to also suggest these edits. Thanks @ashokaditya

Reply:
Looks good, thank you @ashokaditya